ref/net46/Microsoft.Build.Tasks.Core/Microsoft.Build.Tasks.Core.cs

```diff
@@ -2545,6 +2545,7 @@ public enum InstanceState : uint
     {
         Complete = (uint)4294967295,
         Local = (uint)1,
+        NoErrors = (uint)8,
```

Comment:
`VisualStudio.Microsoft.VisualStudio.Setup.Configuration.Interop` works by embedding interop types into our assemblies. This type is one of theirs.

I wish it wasn't public in our assembly but I don't know of a way to avoid it.